perses/rjmc/geometry.py

```diff
@@ -2054,7 +2077,10 @@ def get_modified_system(self):
         """
         return self._growth_system
 
-    def _determine_extra_torsions(self, torsion_force, reference_topology, growth_indices):
+    def _determine_extra_torsions(self,
```

Comment:
`_determine_extra_torsions` allows for rotamer torsions for ring closures and improper torsions for chirality; TODO: add biasing torsions for bond stereochemistry!!!

Reply:
add TODO to the actual code!